### ISSUE TITLE
Collaborate with formulae.brew.sh

### ISSUE THREAD

**Issue by MikeMcQuaid** [zedtux/brewformulas.org#221]
Hey @zedtux! Thanks for all your work on this. Now that formulae.brew.sh is an official project, using GitHub Pages and with a JSON API I was wondering if you were interested in collaborating at all.

**Comment by zedtux** [comment]
Hey @MikeMcQuaid. Thank you for reaching me.

How could I collaborate with you on this projet?

**Comment by MikeMcQuaid** [comment]
@zedtux I'm not sure! I'd love to get to a place where the functionality of `formulae.brew.sh` entirely supersedes brewformulas.org and you might want to help us maintain that instead!

**Comment by MikeMcQuaid** [comment]
So the question for you I guess is: what would we need to do to get to that point?

**Comment by zedtux** [comment]
First of all, I'm completely fine to get brewformulas.org superseded by formulae.brew.sh but as you're using Github pages, you are very limited (no search engine, trending graphs and so on) so a lot of features will be missing, if you stay like this.

I need to know more from you about where you want to go with formulae.brew.sh?
_Honestly I'm a bit confused by your request as I was thinking you opened this issue having a plan in head._

**Comment by zedtux** [comment]
@MikeMcQuaid brewformulas.org is a kind of sandbox for me when I want to test things (last thing was bootstrap 4), and it is my most popular project, don't want to *just* shut it down.

I would love to participate to something and collaborate on a project, not shutting down mine and come to work on yours.

But more importantly I'm a developer, I want to develop things, and using the Github pages with an Algolia instance to implement a search isn't developing for me.
(Playing with Travis CI in order to get a fresh index sounds like a sys admin job to me).

Don't get me wrong, I don't want to be rude too. I'm just saying that I want to develop **interesting** stuff to serve people's need.

To summarise what I'm trying to say : You want to build a cool project with a nice technology stack and plenty of useful and well designed features, I'm your man. If you want to limit the time spent on this project as much as possible, uses small tools to build it, then my answer is no and you can close this issue.

I'm still open to discuss further this topic with you if you'd like to!

**Comment by MikeMcQuaid** [comment]
> If you want to limit the time spent on this project as much as possible, uses small tools to build it, then my answer is no and you can close this issue.

Yep, that's the case. No worries, thanks!
